applications/vessel-history/src/features/settings/components/SettingsComponents.module.css

```diff
@@ -0,0 +1,24 @@
+.settingsFieldsContainer {
+  margin: 15px;
```

Comment:
Try to use the variables that we currently have defined in our [base.css](https://github.com/GlobalFishingWatch/frontend/blob/develop/packages/ui-components/src/base.css#L59) for spacing and sizes whenever is possible:
```

  /* SIZES */
  --size-XS: 2rem;
  --size-S: 2.8rem;
  --size-M: 4rem;
  --size-L: 6rem;
  --space-XS: 0.5rem;
  --space-S: 1rem;
  --space-M: 2rem;
  --space-L: 4rem;
```

Reply:
So I need to modify the design? XS is 20px

Reply:
No, I'm suggestion to use the variables from the design that best match your needs. Juanca @satellitestudiodesign what do you think?